File: C++/AIZU/HelloWorld/watch.cpp
#include <iostream>
//#include <array>

using namespace std;

char watch(int seconds) {
  int minutes;
  int hours;

  hours = seconds / 3600;
  minutes = (seconds - hours * 3600) / 60;
  seconds = seconds % 60;

  printf("%d:%d:%d", hours, minutes, seconds);
}

int main() {
  watch(46979);
  return 0;
}

File: scala/AIZU/watch.scala
object Watch {
  def main(args: Array[String]) {
    output(46979)
  }

  def output(seconds: Int): Unit = {
    val hours = seconds / 3600
    val minutes = (seconds - hours * 3600) / 60
    val secondso = seconds % 60

    printf("%d:%d:%d", hours, minutes, secondso)
  }
}
